src/core/engines/Cesium/useEngineRef.ts

```diff
@@ -354,6 +354,24 @@ export default function useEngineRef(
         if (!viewer || viewer.isDestroyed()) return;
         viewer.clock.stopTime = JulianDate.fromDate(stop);
       },
+      inViewport: location => {
```

Comment:
This is defined for https://github.com/reearth/reearth-web/pull/399/files#diff-08a1099461e54cd367610369d617e713a98ff05cf0e5eb1060ec3e7ec1843a29R74